### PR TITLE
MAUI-900336-UGVolume3-Content added.

### PR DESCRIPTION
**Pull request by IndumathiR-1995**
Hi @MuniappanSubramanian ,

UG document prepare for the  new features in Picker and Calendar controls.

Regards,
Indumathi R

### CODE REVIEW

File: MAUI/Calendar/commands.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Commands
+
+## ViewChangedCommand
+
+The [ViewChangedCommand]() is invoked whenever the view is navigated to either the previous or next view. Additionally, it is triggered when switching between different calendar views, with the `CalendarViewChangedEventArgs` provided as a parameter.
```
Comment (Jeyasri-Murugan) [inline MAUI/Calendar/commands.md:14]: proper explaination of which event can be used

File: MAUI/DateTimePicker/commands.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+<picker:SfDateTimePicker x:Name="picker"
+                    AcceptCommand="{Binding AcceptCommand}">
+<ContentPage.BindingContext>
+    <local:ViewModel/>
+</ContentPage.BindingContext>					  
+</picker:SfDateTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="3,5,7" %}
+
```
Comment (Jeyasri-Murugan) [inline MAUI/DateTimePicker/commands.md:61]: Add proper image for strike through

File: MAUI/Calendar/commands.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+        // To do your requirement here.
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## SelectionChangedCommand
+
+The [SelectionChangedCommand]() is invoked whenever selection changes in the calendar  changes, with the `CalendarSelectionChangedEventArgs` passed as a parameter.
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/commands.md:47]: Mention which event is associated with this command

File: MAUI/Calendar/commands.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+public class CalendarViewModel
+{
+    public ICommand ViewChangedCommand { get; set; }
+    public CalendarViewModel()
+    {
+        ViewChangedCommand = new Command<CalendarViewChangedEventArgs>(ViewChanged);
+    }
+    private void ViewChanged(CalendarViewChangedEventArgs args)
+    {
+        // To do your requirement here.
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/commands.md:38]: Update Migration also

https://help.syncfusion.com/maui/calendar/migration

File: maui-toc.html [hunk 5]
```diff
@@ -1287,6 +1291,7 @@
             <li><a href="/maui/TimePicker/time-interval">Time Interval</a></li>
             <li><a href="/maui/TimePicker/customizations">Customizations</a></li>
             <li><a href="/maui/TimePicker/events">Events</a></li>
+			<li><a href="/maui/TimePicker/commands">Commands</a></li>
```
Comment (MuniappanSubramanian) [inline maui-toc.html:1294]: As per discussion move commands in events topic

File: MAUI/Calendar/date-restrictions.md [hunk 6]
```diff
@@ -96,4 +96,4 @@ this.Calendar.SelectableDayPredicate = (date) =>
 {% endhighlight %}
 {% endtabs %}
 
-![Month view selectable day predicate in .NET MAUI Calendar.](images/date-restriction/net-maui-selectable-day-predicate.png)
\ No newline at end of file
+![Month view selectable day predicate in .NET MAUI Calendar.](images/date-restriction/net-maui-selectable-day-predicate.jpg)
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/date-restrictions.md:99]: Get image from UX team

File: MAUI/Calendar/events.md [hunk 7]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### DoubleTappedCommand
+
+SfCalendar includes a built-in event called `DoubleTapped` and invokes this event through the [DoubleTappedCommand]() whenever the calendar view is double tapped, with the `CalendarDoubleTappedEventArgs` passed as parameter.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/events.md:286]: The `SfCalendar` includes a built-in event called `DoubleTapped` that is triggered whenever the calendar view is double-tapped. This event can be invoked through the [DoubleTappedCommand](), which passes the `CalendarDoubleTappedEventArgs` as a parameter.

File: MAUI/Calendar/events.md [hunk 8]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### TappedCommand
+
+SfCalendar includes a built-in event called `Tapped` and invokes this event through the [TappedCommand]() whenever the calendar view is tapped, with the `CalendarTappedEventArgs` passed as a parameter.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/events.md:253]: The `SfCalendar` includes a built-in event called `Tapped` that is triggered whenever the calendar view is tapped. This event can be invoked through the [TappedCommand](), which passes the `CalendarTappedEventArgs` as a parameter.

File: MAUI/Calendar/events.md [hunk 9]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
 {% endhighlight %}
 {% endtabs %}
 
+## Commands
+
+### ViewChangedCommand
+
+SfCalendar includes a built-in event called `ViewChanged` and invokes this event through the [ViewChangedCommand]() whenever the view is navigated to either the previous or next view. Additionally, the event is triggered when switching between different calendar views, with the CalendarViewChangedEventArgs provided as a parameter.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/events.md:187]: The `SfCalendar` includes a built-in event called `ViewChanged`, which is triggered whenever the calendar view is navigated to either the previous or next view. This event can be invoked through the [ViewChangedCommand](), and it also triggers when switching between different calendar views. The `CalendarViewChangedEventArgs` is provided as a parameter to this event.

File: MAUI/Calendar/events.md [hunk 10]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### LongPressedCommand
+
+SfCalendar includes a built-in event called `LongPressed` and invokes this event through the [LongPressedCommand]() whenever the calendar view is long pressed, with the `CalendarLongPressedEventArgs` passed as parameter.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/events.md:319]: The `SfCalendar` includes a built-in event called `LongPressed`, which is triggered when the calendar view is long pressed. This event can be invoked through the [LongPressedCommand](), with the `CalendarLongPressedEventArgs` passed as a parameter.

File: MAUI/Calendar/events.md [hunk 11]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### AcceptCommand
+
+SfCalendar includes a built-in event called `ActionButtonClicked` and invokes this event through the [AcceptCommand]() whenever confirm button tapped on calendar.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/events.md:352]: The `SfCalendar` includes a built-in event called `ActionButtonClicked`, which is triggered when the confirm button is tapped on the calendar. This event can be invoked through the [AcceptCommand]().

File: MAUI/Calendar/events.md [hunk 12]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### DeclineCommand
+
+SfCalendar includes a built-in event called `ActionButtonCanceled` and invokes this event through the [DeclineCommand]() whenever the cancel button tapped on calendar.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/events.md:385]: The `SfCalendar` includes a built-in event called `ActionButtonCanceled`, which is triggered when the cancel button is tapped on the calendar. This event can be invoked through the [DeclineCommand]().

Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:385]: Content updated.

File: MAUI/Calendar/events.md (hunk 11 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:352]: Content updated.

File: MAUI/Calendar/events.md (hunk 10 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:319]: Content updated.

File: MAUI/Calendar/events.md (hunk 9 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:187]: Content updated.

File: MAUI/Calendar/events.md [hunk 13]
```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### SelectionChangedCommand
+
+SfCalendar includes a built-in event called `SelectionChanged` and invokes this event through the [SelectionChangedCommand]() whenever selection changes in the calendar  changes, with the `CalendarSelectionChangedEventArgs` passed as a parameter.
+
```
[the remark this answers is not in the document]
Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:220]: Content updated.

File: MAUI/Calendar/events.md (hunk 8 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:253]: Content updated.

File: MAUI/Calendar/events.md (hunk 7 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/events.md:286]: Content updated.

File: MAUI/Calendar/date-restrictions.md (hunk 6 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/date-restrictions.md:99]: UX team provided image attached.

File: maui-toc.html (hunk 5 above)
Comment (IndumathiR-1995) [reply maui-toc.html]: Commands topic moved under Events.

File: MAUI/DatePicker/commands.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+control: SfDatePicker
+documentation: ug
+---
+
+# Commands
+
+## SelectionChangedCommand
+
+The [SelectionChangedCommand]() is triggered whenever the selection index in the picker changes, with the `DatePickerSelectionChangedEventArgs` passed as a parameter.
+
```
[the remark this answers is not in the document]
Comment (IndumathiR-1995) [reply MAUI/DatePicker/commands.md]: Migration updated

File: MAUI/Calendar/commands.md (hunk 4 above)
Comment (IndumathiR-1995) [reply MAUI/Calendar/commands.md]: Migration topic updated.

**Review by MuniappanSubramanian — APPROVED** [review]

**Pull request merged**